lib/controllers/transactions_controller.dart

```diff
@@ -64,11 +74,23 @@ class TransactionController {
   }
 }
 
+// Providers
 final transactionsNotifierProvider =
     StateNotifierProvider<TransactionsNotifier, AsyncValue<List<Map>>>((ref) {
   return TransactionsNotifier(ref.read);
 });
 
+final expenseTransactionsProvider =
+StateNotifierProvider<ExpenseTransactionsNotifier, AsyncValue<List<Map>>>((ref) {
+  return ExpenseTransactionsNotifier(ref.read);
+});
+
+final incomeTransactionsProvider =
```

Comment:
temporary workaround, to just finish today.
need to restructure Tablayout and Transactions